### PR TITLE
Added Blocks struct to WebhookMessage

### LINKED ISSUE #594: Add `Blocks` to WebHooks
We are using the webhooks calls.  We would like to add in the ability to send the new `Block` Messages.   I will PR the update later this week. 

https://github.com/nlopes/slack/blob/50e524da56af7c1b27cc221baaa70cfe3d926804/webhooks.go#L11-L20

### PR DESCRIPTION
**Pull request by Dalabad**
Replaces [PR-595](https://github.com/slack-go/slack/pull/595) and adds the missing tests.
Closes [Issue-594](https://github.com/slack-go/slack/issues/594)

##### Pull Request Guidelines

These are recommendations for pull requests.
They are strictly guidelines to help manage expectations.

##### PR preparation
Run `make pr-prep` from the root of the repository to run formatting, linting and tests.

##### Should this be an issue instead
- [ ] is it a convenience method? (no new functionality, streamlines some use case)
- [ ] exposes a previously private type, const, method, etc.
- [ ] is it application specific (caching, retry logic, rate limiting, etc)
- [ ] is it performance related.

##### API changes

Since API changes have to be maintained they undergo a more detailed review and are more likely to require changes.

- no tests, if you're adding to the API include at least a single test of the happy case.
- If you can accomplish your goal without changing the API, then do so.
- dependency changes. updates are okay. adding/removing need justification.

###### Examples of API changes that do not meet guidelines:
- in library cache for users. caches are use case specific.
- Convenience methods for Sending Messages, update, post, ephemeral, etc. consider opening an issue instead.

### CODE REVIEW

File: webhooks_test.go [hunk 1]
```diff
@@ -63,3 +65,27 @@ func TestPostWebhook_NotOK(t *testing.T) {
...
+
+	msgTwoBlocks := WebhookMessage{Blocks: twoBlocks}
+	assert.Equal(t, 2, len(msgTwoBlocks.Blocks.BlockSet))
+
+	msgJsonTwoBlocks, _ := json.Marshal(msgTwoBlocks)
+	assert.Equal(t, `{"blocks":[{"type":"section","text":{"type":"plain_text","text":"text"}},{"type":"section","text":{"type":"plain_text","text":"text"}}]}`, string(msgJsonTwoBlocks))
+
+	msgNoBlocks := WebhookMessage{Text: "foo"}
+	msgJsonNoBlocks, _ := json.Marshal(msgNoBlocks)
+	assert.Equal(t, `{"text":"foo","blocks":null}`, string(msgJsonNoBlocks))
```
Comment (kanata2) [inline webhooks_test.go]: I think it is better to omit `blocks` property from encoded json if `blocks` is null.

Comment (Dalabad) [reply webhooks_test.go]: The `blocks` property is now omitted if it is empty. Sadly the travis build crashed due to an error, which based on what I see is not related to the latest commit. Could you please restart the build?

**Review by kanata2 — APPROVED** [review]
Thanks! LGTM!

**Comment by Phylu** [comment]
Is there anything missing here for it to be merged?

**Comment by kanata2** [comment]
merged!

**Comment by Phylu** [comment]
Thanks! :)

**Pull request merged**